mongodb_migrations/cli.py

```diff
@@ -76,7 +79,9 @@ def _do_rollback(self):
                     module = __import__(self.migrations[migration_datetime])
                     migration_object = module.Migration(host=self.config.mongo_host,
                                                         port=self.config.mongo_port,
-                                                        database=self.config.mongo_database,
+                                                        database=self.config.mongo_database,                                                        
```

Comment:
same for this line, I can see that there are 8 more extra space in the end of this line

Reply:
sure :)